src/main/java/com/courtcanva/ccfranchise/controllers/FranchiseeController.java

```diff
@@ -38,4 +45,14 @@ public SuburbListGetDto addDutyAreas(@RequestBody @Valid SuburbListPostDto subur
         return franchiseeService.addDutyAreas(suburbListPostDto, franchiseeId);
     }
 
+    @GetMapping("/{franchiseeId}/orders")
+    @ResponseStatus(HttpStatus.OK)
+    public List<OpenOrderGetDto> getFirst10OpenOrders(@PathVariable Long franchiseeId, @RequestParam @NotNull String status){
+        if (OrderStatus.ASSIGNED_PENDING.getStatus().equals(status)) {
```

Comment:
If this api is  only to get  order in ASSIGNED_PENDING  status, why we need the second parameter?

Reply:
this pr feature is getting open orders, but by differentiating by status, this method can be extended to get other orders like accepted orders(Christina's ticket) .

Reply:
Also, I changed method name from `getFirst10OpenOrders `to `getOrders `to make it universal